.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -9,21 +9,20 @@ A clear and concise description of what the bug is.
 **To Reproduce**
 Steps to reproduce the behavior:
 
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+1. Use Tuist X.X.X
+2. Use the attached sample project (to be provided when opening the issue)
+3. Run `…`
+4. See the error
+
+```
+Copy of the full error message
+```
 
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
```

Comment:
I am ok with removing the macOS version but instead, I'd suggest to specify the Xcode version which is quite valuable to know.